src/webGLClient.js

```diff
@@ -316,7 +316,11 @@ WebGLClient.prefetch = function() {
   // Create a fake temporary GL context
   var canvas = document.createElement('canvas');
   var ctx = canvas.getContext('webgl-experimental') || canvas.getContext('webgl');
-  if (!ctx) return;
+  if (!ctx) {
+    //remove run dependency created in proxyWorker
```

Comment:
space after `//`.

also, how about text like "if we have no webGL support, we still notify that prefetching is done, as the app blocks on that" ?